src/test/java/LoginTests.java

```diff
@@ -1,25 +1,21 @@
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.Assert;
-import org.testng.annotations.Test;
-
-import java.time.Duration;
-
-public class LoginTests extends BaseTest {
-    @Test
-    public void loginEmptyEmailPassword() {
-
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LoginPage;
 
-        String url = "https://qa.koel.app/";
+public class LoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage = new LoginPage(driver);
         driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
     }
-}
+    @Test
+    public void validLogin(){
+        loginPage.loginCorrectCred();
+    }
+    @Test
+    public void validLoginOnceLoggedIn(){
+        loginPage.loginCorrectCred();
+    }
+}
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/3eeabd21-1188-4ed7-b81f-17b3eb9d57a7)

Reply:
That is very strange. Locally all my codes passed.